torch/csrc/jit/passes/onnx/fixup_onnx_controlflow.cpp

```diff
@@ -401,5 +461,36 @@ std::vector<Value*> FixupONNXControlflowNode(Node* n, int opset_version) {
   }
 }
 
+void FixupONNXControlflowNodeOutputs(Node* n) {
+  switch (n->kind()) {
+    case ::c10::onnx::Loop: {
+      for (size_t i = 0; i < n->outputs().size(); ++i) {
```

Comment:
Please use `c10::irange`
```suggestion
      for (auto i : c10::irange(n->outputs().size())) {
```

Reply:
thx, updated.